src/mapper/pkg/cloudapi/discovered_intents_api.go

```diff
@@ -0,0 +1,12 @@
+package cloudapi
+
+import (
+	"context"
+	"github.com/Khan/genqlient/graphql"
+)
+
+type CloudAPIImpl struct{}
+
+func (CloudAPIImpl) ReportDiscoveredSourcedIntents(ctx context.Context, client graphql.Client, environment string, source string, intents []IntentInput) (*ReportDiscoveredSourcedIntentsResponse, error) {
```

Comment:
why is this in a separate file? the cloud client is small enough to be contained in a single file